src/pages/index.js

```diff
@@ -33,40 +59,96 @@ const cardFormValidator = new FormValidator(
   constants.cardFormElement
 );
 
-editFormValidator.enableValidation();
-cardFormValidator.enableValidation();
-
-const cardCreator = new Section(
-  {
-    items: constants.initialCards,
-    renderer: createCard,
-  },
-  ".cards__list"
+const avatarFormValidator = new FormValidator(
+  constants.validationSettings,
+  constants.avatarFormElement
 );
 
-cardCreator.renderItems();
+editFormValidator.enableValidation();
+cardFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
 const userProfileInfo = new UserInfo({
   profileName: ".profile__title",
   profileJob: ".profile__subtitle",
+  profileAvatar: ".profile__image",
+  isLiked: ".card__like-button",
 });
 
+api
+  .getUserInfo()
+  .then((data) => {
+    console.log(">>USER DATA:", data);
+    userProfileInfo.setUserInfo({
+      profileName: data.name,
+      profileJob: data.about,
+      profileAvatar: data.avatar,
+      isLiked: data.isLiked,
```

Comment:
It has no effect here. It's not used, actually

And `data.isLiked` doesn't exist